uri/path_test.go

```diff
@@ -36,18 +36,18 @@ func TestPathRoundtrip(t *testing.T) {
 			wantURI:      "file:///tmp/path/",
 			wantFilepath: "/tmp/path",
 		},
-		{
-			desc:         "encoded path",
-			raw:          "/tmp/file%20with%20space",
-			wantURI:      "file:///tmp/file%20with%20space",
-			wantFilepath: "/tmp/file with space",
-		},
```

Comment:
This case doesn't round trip (it removes encoding on the second pass), but it is valid at all points.